sycl/tools/sycl-xocc/bin/sycl-xocc

```diff
@@ -184,8 +184,9 @@ cp "$INPUT_FILE_NAME" "${TMP_PATH}_${SOURCE_FILE_NAME}_kernels.bc"
 # modified from the upstream implementation to take this argument to avoid
 # adding a new LLVM Target that will largelly be unused right now..
 $OPT --sycl-xocc -preparesycl -globaldce -inline -infer-address-spaces -globaldce -O3 -globaldce\
-  -flat-address-space=0 -globaldce -inSPIRation -globaldce -kernelPropGen \
+  -flat-address-space=0 -globaldce  -globaldce -kernelPropGen \
   --sycl-kernel-propgen-output $KERNELPROP \
+  #-inSPIRation \
```

Comment:
I wonder what a comment in a continuation line really means...
I guess you can remove it anyway.